core/trino-main/src/test/java/io/trino/operator/window/WindowAssertions.java

```diff
@@ -54,6 +54,38 @@ public final class WindowAssertions
             "    (CAST(NULL AS BIGINT), CAST(NULL AS VARCHAR), '1995-07-16')\n" +
             ") AS orders (orderkey, orderstatus, orderdate)";
 
+    private static final String VALUES_WITH_NAN = "" +
```

Comment:
Use text blocks for these multiline strings.

Reply:
Done in https://github.com/trinodb/trino/pull/21082